[2024-08-27T18:20:19.083+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a permission to 509
[2024-08-27T18:20:19.083+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a'
[2024-08-27T18:20:19.112+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a permission to 509
[2024-08-27T18:20:19.112+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a'
[2024-08-27T18:20:19.140+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T18:20:19.150+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: part_a_currency_exchange_dag.calculate_exchange_rate_change_part_a scheduled__2024-08-27T17:00:00+00:00 [queued]>
[2024-08-27T18:20:19.159+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: part_a_currency_exchange_dag.calculate_exchange_rate_change_part_a scheduled__2024-08-27T17:00:00+00:00 [queued]>
[2024-08-27T18:20:19.160+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-08-27T18:20:19.175+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): calculate_exchange_rate_change_part_a> on 2024-08-27 17:00:00+00:00
[2024-08-27T18:20:19.180+0000] {standard_task_runner.py:57} INFO - Started process 236 to run task
[2024-08-27T18:20:19.182+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'part_a_currency_exchange_dag', 'calculate_exchange_rate_change_part_a', 'scheduled__2024-08-27T17:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rate_dag.py', '--cfg-path', '/tmp/tmp5hjctt5j']
[2024-08-27T18:20:19.184+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask calculate_exchange_rate_change_part_a
[2024-08-27T18:20:19.199+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T18:20:19.238+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a permission to 509
[2024-08-27T18:20:19.239+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=part_a_currency_exchange_dag/run_id=scheduled__2024-08-27T17:00:00+00:00/task_id=calculate_exchange_rate_change_part_a'
[2024-08-27T18:20:19.241+0000] {task_command.py:410} INFO - Running <TaskInstance: part_a_currency_exchange_dag.calculate_exchange_rate_change_part_a scheduled__2024-08-27T17:00:00+00:00 [running]> on host 08c83ac7a73b
[2024-08-27T18:20:19.253+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-27T18:20:19.327+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='part_a_currency_exchange_dag' AIRFLOW_CTX_TASK_ID='calculate_exchange_rate_change_part_a' AIRFLOW_CTX_EXECUTION_DATE='2024-08-27T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-27T17:00:00+00:00'
[2024-08-27T18:20:19.333+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-08-27T18:20:19.342+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=part_a_currency_exchange_dag, task_id=calculate_exchange_rate_change_part_a, execution_date=20240827T170000, start_date=20240827T182019, end_date=20240827T182019
[2024-08-27T18:20:19.358+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-08-27T18:20:19.375+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
